src/include/block.h

```diff
@@ -174,6 +174,9 @@ struct __wt_block_ckpt {
     WT_EXTLIST ckpt_discard; /* Checkpoint archive */
 };
 
+#define WT_BLOCK_INVALID_PAGE_ID 0 /* Invalid page ID, e.g., if it's not allocated. */
+#define WT_BLOCK_MIN_PAGE_ID 100   /* Minimum page ID that can be used for user data. */
+
```

Comment:
Are we designating any particular page id (like 1) for root?  That was an idea that was floating around in disagg design discussions.

Reply:
Not currently. This has an emergent behavior that 100 is the root page, but it will not stay like that if we decide to use different page IDs for all pages in a split (as we've been also discussing). Is this worth ensuring that we always have the same page ID for the root? In which case would it be helpful?

Reply:
I don't think it's necessary to have a fixed root page id.  It's probably convenient for debugging though.